src/libcore/macros.rs

```diff
@@ -168,8 +168,14 @@ macro_rules! try {
     })
 }
 
-/// Use the `format!` syntax to write data into a buffer of type `&mut Write`.
-/// See `std::fmt` for more information.
+/// Use the `format!` syntax to write data into a buffer.
+///
+/// The buffer must be a `&mut [Write][write]`.
+///
+/// See [`std::fmt`][fmt] for more information on format syntax.
+///
+/// [fmt]: fmt/index.html
+/// [write]: io/trait.Write.html
```

Comment:
So, we gave up on making rustdoc do crosslinking? Who’s going to check these links do not break in the future?